app/src/main/java/com/xmartlabs/template/helper/GeneralErrorHelper.java

```diff
@@ -67,32 +67,21 @@ public GeneralErrorHelper() {
     BaseProjectApplication.getContext().inject(this);
   }
 
-  private void logCrashlyticsError(ServiceExceptionWithMessage exceptionWithMessage) {
+  private void logError(ServiceExceptionWithMessage exceptionWithMessage) {
     String url = ServiceHelper.getUrl(exceptionWithMessage.getResponse().raw());
     int resultCode = exceptionWithMessage.getCode();
     String headers = exceptionWithMessage.getResponse().headers().toString();
     String body = exceptionWithMessage.getErrorBody();
-    Crashlytics.setString(CRASHLYTICS_KEY_URL, url);
```

Comment:
you are losing the keys, provide a way of sending they to the logger! Maybe you can inject the logger, and make something with that.